PhysicsTools/NanoAOD/plugins/LeptonInJetProducer.cc

```diff
@@ -91,6 +91,8 @@ void LeptonInJetProducer<T>::produce(edm::StreamID streamID, edm::Event &iEvent,
       fastjet::PseudoJet p(d->px(), d->py(), d->pz(), d->energy());
       lClusterParticles.emplace_back(p);
     }
+    ele_pfmatch_index = -1;
```

Comment:
the two variables can be created here, removing the definitions lines 79-80

Reply:
I prefer the current one since define the variables every time in the loop may use more memory, though the difference could be small since they are "int".